tests/framework/cli/test_starters.py

```diff
@@ -5,6 +5,7 @@
 import shutil
 from pathlib import Path
 
+import anyconfig
```

Comment:
@ankatiyar just got rid of `anyconfig` in #2692 , can we use `toml` or `omegaconf` here instead as well?